docs/docusaurus.config.js

```diff
@@ -59,6 +59,14 @@ module.exports = {
                         to: '/self-host/self-host-lightdash',
                         from: ['/self-host'],
                     },
+                    {
+                        to: 'https://intercom.help/lightdash-faqs/en/collections/8439498-faqs',
+                        from: ['/help-and-contact/faqs/faqs']
```

Comment:
Looks like the version we use doesn't allow redirects to external link

> 9:00:43 AM: [ERROR] ValidationError: "redirects[8].to" is not a valid pathname. Pathname should start with slash and not contain any domain or query string.

They added this feature in v3
https://github.com/facebook/docusaurus/pull/9171

Reply:
ah, fair enough. I'll do that then!

Reply:
sorry all!